pydicom/dataelem.py

```diff
@@ -567,7 +591,7 @@ def repval(self):
             repVal = repr(self.value)  # will tolerate unicode too
         return repVal
 
-    def __unicode__(self):
+    def __unicode__(self) -> str:
```

Comment:
I think `__unicode__` is not used anymore in Python 3, so can be removed.

Reply:
Should I just remove it now?

Reply:
Removed